packages/storybook-aem/storybook/middleware.js

```diff
@@ -1,12 +1,12 @@
-const proxy = require('http-proxy-middleware')
+const { createProxyMiddleware } = require('http-proxy-middleware')
```

Comment:
@jzeltman This only works if I change how the proxy get's required. Do we have some versioning difference?

Reply:
There is a versioning difference. They adjusted the default export. Need to update our dependency version I think.